app/src/shared/components/Tile/ImageContainer.jsx

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import React, { type Node } from 'react'
+import styled from 'styled-components'
+import ReactImage from 'react-image'
+import Skeleton from '$shared/components/Skeleton'
+import UnstyledLogo from '$shared/components/Logo'
+
+type Props = {
+    alt?: ?string,
+    children?: Node,
+    height?: any,
+    ratio?: string,
+    skeletonize?: boolean,
+    src?: ?string,
+}
+
+const Root = styled.div`
+    border-radius: 2px;
+    border-radius: ${({ theme }) => theme.borderRadius};
+    overflow: hidden;
+    position: relative;
+
+    svg {
+        display: block;
+    }
+`
+
+const Logo = styled(UnstyledLogo)`
+    height: auto;
+    left: 50%;
+    max-width: 32%;
+    position: absolute;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    width: 80px;
+`
+
+export const Image = styled(ReactImage)`
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+
+    img& {
+        object-fit: cover;
+    }
+`
+
+const Placeholder = styled.div`
+    background-image: linear-gradient(135deg, #0045FF 0%, #7200EE 100%);
+`
+
+const ImageContainer = ({
+    alt = '',
+    height,
+    src,
+    ratio = '3:2',
+    children,
+    skeletonize,
+    ...props
+}: Props) => (
+    <Root {...props}>
+        {src != null && (
+            skeletonize ? (
+                <Image as={Skeleton} block />
```

Comment:
Here's a nice use case for `styled-components`. `Image` acts both as a `Skeleton` (both props-driven and as a loader) and as a `Placeholder`, and of course as the actual `<Image />`. Managed to use it 4 times around here! 💃